pages/herosection.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Slideshow from '../components/Slideshow';
+
+function herosection() {
```

Comment:
This is not a page? and only returns a div with the component <Slideshow>. Is it not better to just add the component in index.js instead?

Reply:
moved herosection to components.